mitosheet/mitosheet/api/get_dataframe_as_excel.py

```diff
@@ -24,20 +24,20 @@ def get_dataframe_as_excel(params: Dict[str, Any], steps_manager: StepsManagerTy
     # We write to a buffer so that we don't have to save the file
     # to the file system for no reason
     buffer = io.BytesIO()
-    with pd.ExcelWriter(buffer) as writer:
+    with pd.ExcelWriter(buffer, engine="openpyxl") as writer:
         for sheet_index in sheet_indexes:
-            # First, write the tab in the Excel file
+            # Get the dataframe and sheet name
             df = steps_manager.dfs[sheet_index]
             df_name = steps_manager.curr_step.df_names[sheet_index]
-            df.to_excel(
-                writer, 
-                sheet_name=get_df_name_as_valid_sheet_name(df_name),
-                index=False
-            )
+            sheet_name = get_df_name_as_valid_sheet_name(df_name)
 
-            # Adding formatting is a pro feature
-            if is_pro_user:
-                add_formatting_to_excel_sheet(writer, steps_manager, sheet_index)
+            # Write the dataframe to the sheet
+            df.to_excel(writer, sheet_name, index=False)
+
+            # Add formatting to the sheet for pro users
+            format = steps_manager.curr_step.df_formats[sheet_index]
+            if is_pro_user and format['headers']: 
```

Comment:
This if returns false if you're not Pro, so the tests aren't actually testing this code path. 

We can fix this by adding a line that says `if (is_pro_user or is_running_test()) and ...`

Reply:
Ah true I added that but it must have gotten lost between git stashes!